docs/src/components/navigation/header-nav.tsx

```diff
@@ -0,0 +1,44 @@
+'use client';
+
+import Button from '../button';
+import { KeystaticLogo } from './keystatic-logo';
+import { SocialLinks } from './social-links';
+import { MobileNav } from './mobile-nav';
+import { usePathname } from 'next/navigation';
+
+export type NavProps = {
+  navigationMap?: {
+    groupName: string;
+    items: {
+      label: string;
+      href: string;
+      title: string | undefined;
+    }[];
+  }[];
+};
+
+export function HeaderNav({ navigationMap }: NavProps) {
+  const pathname = usePathname();
+  const isDocsNav = pathname?.includes('/docs');
```

Comment:
Maybe a bit pedantic but what do you think about `pathname?.startsWith('/docs')? Just so we don't get something like `/templates/docs-template` etc?

Reply:
Good thinking

Reply:
Great catch – thank you!